tools/hangar/__snapshots__/plugins.ts.snap

```diff
@@ -171,7 +171,7 @@ exports[`Plugin examples > AWS target plugins > permission-boundary.js 1`] = `
   "terraform": {
     "backend": {
       "local": {
-        "path": "/Users/hasan/repos/monada/wing/tools/hangar/tmp/terraform.root.tfstate",
+        "path": "/Users/chrisr/dev/wing3/tools/hangar/tmp/terraform.root.tfstate",
```

Comment:
whoopsie, didn't catch this in the last pr

@hasanaburayyan

Reply:
This should be caught by CI, since the GitHub runner would probably generate a snapshot with a different value, no?

Reply:
😢 https://github.com/winglang/wing/blob/ecd00497bae6d6c6cb2ac5c88a79b67388cafbf8/tools/hangar/vitest.config.ts#L8

I'll do a cleanup PR, it's fine to merge this one